docs/source/quickstart/local-subgraphs.mdx

```diff
@@ -11,28 +11,75 @@ Our subgraphs can use any GraphQL server library that supports Apollo Federation
 
 > If you're using a GraphQL server besides Apollo Server, consult its documentation to learn how to configure it for use as a subgraph.
 
-If you have an existing GraphQL API that uses Apollo Server, you can use that server as a subgraph with the `@apollo/subgraph` library. You can get started with [this example server](https://github.com/apollographql/docs-examples/tree/main/apollo-server/v3/getting-started) (which is not yet federated).
+If you have an existing GraphQL API that uses Apollo Server, you can use that server as a subgraph with the `@apollo/subgraph` library. You can get started with [this example server](https://github.com/apollographql/docs-examples/blob/main/apollo-server/v4/getting-started) (which is not yet federated).
 
-First, install the latest version of the `@apollo/subgraph` library in your project:
+First, install the latest version of `@apollo/subgraph` and `graphql-tag` into your project:
 
 ```shell
-npm install @apollo/subgraph
+npm install @apollo/subgraph graphql-tag
 ```
 
+Import the `gql` tag into the file where you create your schema (`index.ts` for our example):
```

Comment:
This is fine to leave this as-is, but could be consolidated into the step below.

Reply:
I like that! Update to: 
> Import the `gql` tag into the file where you create your schema (`index.ts` for our example), and wrap your schema in the that tag, like so:
> 
> ```ts title="index.ts"
> import gql from 'graphql-tag'; // preserve-line
> 
> const typeDefs = gql`
>   type Book {
>    etc.
>  }
>`
>```